### PR TITLE
Wip master outline

### PR DESCRIPTION
**Pull request by drakeltheryuujin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,171 @@
...
 ## Introduction
 
-## SWBAT 1
+Iterating over arrays is a very common task we will find ourselves
+performing as developers. Using `for` loops, can get tedious after
+a while. Rather than using `for` loops and nesting to process data
+in lists and collections, you can take advantage a method like `map()`
+to better organize your logic into building blocks of functions,
+and then chain them to create more readable and understandable
+implementations.
```
Comment (jenmyers) [inline README.md]: I feel we could make this paragraph flow a little better. For example: "As developers, we find ourselves responsible for all sorts of common but tedious tasks, such as iterating over arrays. While a `for` loop or nesting in lists and collections will complete the task, we can take advantage of a method like `map()` to organize and optimize our logic into building blocks of functions, which we can then chain together to create more readable and understandable implementations." It's all there, but smoothing it out will help students slide into the lesson more easily.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,171 @@
...
+//=> patroni = ["Stag", "Jack Russell Terrier", "Otter", "Horse"];
+```
+
+With `map()` we're passing a function _as an argument_. Yes, arguments can be
+things like `Number` or `String`...but, in JavaScript, **can also** be
+**work**. Very few other programming languages allow that! That's worthy of a
+Keanu "Whoa."
+
+![Whoa](https://media.giphy.com/media/ALZ1PPM20REZ2/giphy.gif)
+
```
Comment (jenmyers) [inline README.md]: I think we all agreed we weren't going heavy on gifs moving forward ... I don't think this necessarily has to be taken out, but I don't think we necessarily need it either. I like Keanu! But I also wonder if the reference is a little dated.

File: README.md [hunk 3]
```diff
@@ -1,16 +1,171 @@
...
+   { name: 'Sonny', modes: 5, isActivated: true },
+   { name: 'Baymax', modes: 3, isActivated: true }
+ ]
+*/
+```
+
+With  the native `map()` function that is a property
+of `Array`'s prototype. It gives us the exact same result! Now that we know how
+map is implemented, it holds no more secrets for us! We can discard our own `map()`
+function and just use the `.map()` property on arrays. Sweet!
```
Comment (jenmyers) [inline README.md]: This applies the entire main body of the text: it's all structured well and clearly expressed, but I think this is an occasion that our readability tool would be helpful. There are a lot of longer words in here that I wonder couldn't be switched out for simpler ones. I would recommend just going through with that in mind and seeing if anything can be simplified.

File: README.md (hunk 2 above)
Comment (drakeltheryuujin) [reply README.md]: This probably was leftover from porting it over. I agree that gifs (basically memes), unless actually relevant to the learning process, are unnecessary.

**Pull request merged**